### ISSUE TITLE
Search option in schedule builder

### ISSUE THREAD

**Issue by spicerackk** [vexorian/dizquetv#19]
I am picturing a simple search bar that would allow you to search for a show that  you know might have episodes or other content you don't want. 

An example could be Pokemon, in which I have the movies in the "Specials" folder in the TV Library. Currently I have to go through manually and remove any files that say "Pokemon S00EXX, but the search bar could allow me to narrow the schedule down to what I need by typing "Pokemon S00" in the search field, with a remove all button.

**Comment by psybertech** [comment]
add to this one a bit...
removing/deleting things from a channel...

It would be nice to have a separate list of shows/seasons for items already in the channel (grouped if tv/season) and a remove/delete button.

or add a delete button from the adding show selection (right now, it shows the '+' to add, but if that entry is already added, show a '-' button to remove it.

I only use this for TV so far, but the concept should work well for movies as well.

Love this so far! =)

**Comment by MoRBiD-MaNGLeR** [comment]
> An example could be Pokemon, in which I have the movies in the "Specials" folder in the TV Library. Currently I have to go through manually and remove any files that say "Pokemon S00EXX, but the search bar could allow me to narrow the schedule down to what I need by typing "Pokemon S00" in the search field, with a remove all button.

Since we are talking about Schedule Builder functionality as it relates to Specials, rather than create a new Enhancement Request, I thought I'd add to this one:

When adding a series, at the series level, it would be great to have an option next to the "+" for "Add All Except Specials".  This would save time in not having to remove the Specials after adding the series (or would prevent having to add each Season individually to avoid adding the Specials).

**Comment by psybertech** [comment]
> When adding a series, at the series level, it would be great to have an option next to the "+" for "Add All Except Specials". This would save time in not having to remove the Specials after adding the series (or would prevent having to add each Season individually to avoid adding the Specials).

Yeah, the specials can be a bit annoying and I have had to recreate a few channels already after seeing they where there... need to be more careful when creating. This would be a great feature (and the ability to delete stuff in bulk without having to recreate would rock too).

j

**Comment by MoRBiD-MaNGLeR** [comment]
@spicerackk @psybertech There are now Tools buttons for Remove Duplicates, Remove Flex, Remove Specials and Remove All.  Do you feel this request is handled by the mentioned buttons?  If not, please clarify what part of the request remains and include a detailed use case.

**Anything that hasn't already been mentioned should be added to a New Issue.**

**Comment by psybertech** [comment]
Love the new remove specials button!
Works for me!

**Comment by MoRBiD-MaNGLeR** [comment]
@spicerackk How about you?

**Comment by 19wolf** [comment]
I would like a search option in the schedule builder, if I want to add particular episodes or shows that share a name but aren't the same (scooby-doo, stargate, star trek, etc).
